src/components/TeamForm/TeamForm.js

```diff
@@ -28,20 +30,26 @@ class TeamForm extends Component {
     };
     this.state = {
       address: props.team.address,
-      name: props.team.name
+      name: props.team.name,
+      voteDuration: props.team.vote_duration
     };
   }
 
   handleChange = field => event => this.setState({ [field]: event.target.value });
 
+  handleKeyPress = field => event => field === 'voteDuration' ? onlyAllowNonZeroDigits(event) : null;
+
   handleSubmit = (event) => {
+    // TODO: add error if null value is submitted for vote duration and prevent submission
     event.preventDefault();
     this.props.updateTeam(Object.assign({}, this.state, this.props.center));
   };
 
   render() {
     const { team } = this.props;
     const { name } = this.state;
+    const voteDuration = this.state.voteDuration;
```

Comment:
Can be combined into line 50, e.g. `const { name, voteDuration } = this.state;`.